src/components/Card/CardVariants.ts

```diff
@@ -0,0 +1,136 @@
+  outline: {
+    background: theme.pallete.surface.main,
+    border: `1px solid ${theme.pallete.gray.c80}`,
+    borderRadius: `${theme.radius.popper}px`,
+    '&[data-invalid="true"]': {
+      borderColor: theme.pallete.status.danger.main,
+    },
+    '&[data-disabled="true"]': {
+      borderColor: theme.pallete.gray.c80,
+      background: theme.pallete.surface.main,
+    },
+  } as ExternalStyles,
+  plain: {
+    background: theme.pallete.surface.main,
+    borderRadius: `${theme.radius.popper}px`,
+    '&[data-invalid="true"]': {
+      background: theme.pallete.status.danger.c90,
+    },
+    '&[data-disabled="true"]': {
+      borderColor: theme.pallete.gray.c80,
+      background: theme.pallete.surface.main,
+    },
+  } as ExternalStyles,
+})
+
+export const createClickableStyles = (theme: Theme): ExternalStyles => ({
+  ...theme.typography.variant('main'),
+  cursor: 'pointer',
+  textAlign: 'initial',
+  '&[data-variant="outline"], &[data-variant="plain"]': {
+    borderColor: theme.pallete.gray.c60,
+  },
+  ':not(:disabled):hover': {
+    background: hexToRGB(theme.pallete.gray.c50, 0.1),
+    '&[data-variant="float"]': {
+      boxShadow: theme.shadows.outer[160],
+    },
+  },
+  ':focus': {
+    outline: 'none',
+    boxShadow: focusBoxShadow(theme),
+    '&[data-variant="float"]': {
+      boxShadow: `${theme.shadows.outer[40]}, ${focusBoxShadow(theme)}`,
+    },
+  },
+  ':not(:disabled):not(:active):hover:focus': {
+    '&[data-variant="float"]': {
+      boxShadow: `${theme.shadows.outer[160]}, ${focusBoxShadow(theme)}`,
+    },
+  },
+  ':not(:disabled):active': {
+    boxShadow: theme.shadows.inner['10'],
+    '&[data-variant="float"]': {
+      // use zero box shadow to keep transition smooth
+      boxShadow: `0 0 0 0 rgba(0, 0, 0, 0), 0 0 0 0 rgba(0, 0, 0, 0), 0 0 0 0 rgba(0, 0, 0, 0), ${theme.shadows.inner['10']}`,
+    },
+  },
+  '&[data-selected="true"]': {
+    background: theme.pallete.primary.c90,
+    borderColor: theme.pallete.primary.main,
+    ':not(:disabled):hover': {
+      background: theme.pallete.primary.c80,
+    },
+    '&[data-invalid="true"]': {
+      background: theme.pallete.status.danger.c90,
+    },
+  },
+  '&[data-invalid="true"]': {
+    borderColor: theme.pallete.status.danger.main,
+    ':not(:disabled):hover': {
+      background: theme.pallete.status.danger.c80,
+    },
+    ':focus': {
+      outline: 'none',
+      boxShadow: focusBoxShadow(theme, 'danger'),
+    },
+    ':not(:disabled):active': {
+      boxShadow: theme.shadows.inner['10'],
+    },
+    ':not(:disabled):not(:active):hover:focus': {
+      '&[data-variant="float"]': {
+        boxShadow: `${theme.shadows.outer[160]}, ${focusBoxShadow(theme, 'danger')}`,
+      },
+    },
+  },
+  ':disabled': {
+    borderColor: theme.pallete.gray.c80,
+    '&[data-selected="true"]': {
+      borderColor: theme.pallete.primary.c80,
+    },
+    '&[data-invalid="true"]': {
+      borderColor: theme.pallete.gray.c80,
```

Comment:
Is it correct to repeat with the definition on the line 123?

Reply:
really wasn't needed, removed! :heavy_check_mark: